README.md

```diff
@@ -99,8 +99,9 @@ COMMANDS:
      ps                      Display your application containers
      scale, s                Scale your application instantly
      restart                 Restart processes of your app
-     force-https
-     sticky-session
+     force-https             Enable/Disable automatic redirection of traffic to HTTPS for your application
+     sticky-session          Enable/Disable sticky sessions for your application
```

Comment:
These lines are just copy-paste of `scalingo help`. Hence you need to update the `Description` attributes of `force-https` and `sticky-session`. This is not just a README update.

Reply:
Do you want me to keep the README description or the command description ?
#### Example for force-https :
Command's description : 
`When enabled, this feature will automatically redirect HTTP traffic to HTTPS for all domains associated with this application.`

**VS**

README description :
`Enable/Disable automatic redirection of traffic to HTTPS for your application`

Reply:
Oh sorry, after double checking I think this is the `Usage` attribute that should be updated with the same text you currently have here (i.e. "Enable/Disable automatic redirection of traffic to HTTPS for your application")

Reply:
Oh okey I get it. Will do.